spec/units.yaml

```diff
@@ -224,3 +224,15 @@ units:
     label: beats per minute
     description: Number of human heart beats per minute
     domain: frequency
+  Ohm:
+    label: resistance
+    description: Electrical resistance measured in ohms
+    domain: electric resistance
+  Wh/km:
```

Comment:
This one is quite similar to `kWh/100km` that we already have, but not use in standard catalog. Are both needed? Is one of them more frequently used when stating/displaying values? Or should this one better replace the old one?

Reply:
I did some googling to see how widespread `Wh/km` and `kWh/100km` are. It seems that both of them are quite frequently used. Maybe kWh/100km seems to more frequent in Europe, Wh/km more frequent in US. As both are quite commonly used in the industry I think it makes sense to have both in the unit file for convenience. (But when/if we are to add signals to the standard catalog we must select one of them as the preferred one, but that is a later question.)

**Wh/km**

https://ev-database.org/cheatsheet/energy-consumption-electric-car
https://elbilsmagasinet.se/testad/rackviddstest/
https://insideevs.com/news/567953/2022-ford-mache-range-efficiency/ (also Wh/mi)

**kWh/100km** 

https://www.entega.de/blog/elektroauto-verbrauch/
https://www.vattenfall.de/infowelt-energie/elektroauto-stromverbrauch
https://www.tesla.com/sv_se/support/power-consumption